packages/react-table/src/components/TableComposable/examples/Table.md

```diff
@@ -60,33 +60,33 @@ import styles from '@patternfly/react-styles/css/components/Table/table';
 import { Dropdown as DropdownNext, DropdownList } from '@patternfly/react-core/next';
 
 
-## TableComposable examples
+## Table examples
 
-The `TableComposable` component differs from the regular `Table` component, in that it allows you to compose the table by nesting the relevant `Thead`, `Tbody`, `Tr`, `Th` and `Td` components within it. For a less declarative and more implicit approach, use the `Table` component instead.
+This `TableComposable` component differs from the deprecated `Table` component, in that it allows you to compose the table by nesting the relevant `Thead`, `Tbody`, `Tr`, `Th` and `Td` components within it. For a less declarative and more implicit approach, use the `Table` component instead.
```

Comment:
is it still accurate to say "This `TableComposable` component" rather than "This `Table` component"? The general bulleted notes below reference the Tablecomposable, too.